utilities/rpp-unittests/HOST_NEW/testAllScript_now.sh

```diff
@@ -0,0 +1,433 @@
+#!/bin/bash
+
+
+
+
+
+# <<<<<<<<<<<<<< DEFAULT SOURCE AND DESTINATION FOLDERS (NEED NOT CHANGE) >>>>>>>>>>>>>>
+
+cwd=$(pwd)
+test_type=$4
+
+# Input Images - Single image (224 x 224)
+# DEFAULT_SRC_FOLDER_1="$cwd/../TEST_IMAGES/single_image_224x224_src1"
+# DEFAULT_SRC_FOLDER_2="$cwd/../TEST_IMAGES/single_image_224x224_src2"
+
+# Input Images - Two images (224 x 224)
+# DEFAULT_SRC_FOLDER_1="$cwd/../TEST_IMAGES/two_images_224x224_src1"
+# DEFAULT_SRC_FOLDER_2="$cwd/../TEST_IMAGES/two_images_224x224_src2"
+
+# Input Images - Three images (224 x 224)
+DEFAULT_SRC_FOLDER_1="$cwd/../TEST_IMAGES/three_images_224x224_src1"
+DEFAULT_SRC_FOLDER_2="$cwd/../TEST_IMAGES/three_images_224x224_src2"
+
+# Input Images - Two images (mixed size)
+# DEFAULT_SRC_FOLDER_1="$cwd/../TEST_IMAGES/two_images_mixed_src1"
+# DEFAULT_SRC_FOLDER_2="$cwd/../TEST_IMAGES/two_images_mixed_src2"
+
+# Output Images
+mkdir "$cwd/../OUTPUT_IMAGES_HOST_NEW"
+DEFAULT_DST_FOLDER="$cwd/../OUTPUT_IMAGES_HOST_NEW"
+
+# for logging
+if [ $test_type -eq 1 ]; then
+    rm -rvf "$cwd/../OUTPUT_PERFORMANCE_LOGS_HOST_NEW"/*
+    mkdir "$cwd/../OUTPUT_PERFORMANCE_LOGS_HOST_NEW"
+    LOGGING_FOLDER="$cwd/../OUTPUT_PERFORMANCE_LOGS_HOST_NEW"
+fi
+
+
+# Images for unique functionalities
+DEFAULT_FAST_CORNER_DETECTOR_IMAGES="$cwd/../TEST_IMAGES/fast_corner_detector"
+DEFAULT_HARRIS_CORNER_DETECTOR_IMAGES="$cwd/../TEST_IMAGES/harris_corner_detector"
+DEFAULT_HOUGH_LINES_IMAGES="$cwd/../TEST_IMAGES/hough_lines"
+DEFAULT_HOG_IMAGES="$cwd/../TEST_IMAGES/hog"
+num_iterations=100
```

Comment:
Remove all hardcoded variables. The num_iterations should be passed a parameter from the python script